src/react-app/src/index.tsx

```diff
@@ -1,11 +1,17 @@
-import React from 'react';
+import React, { Suspense } from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import { registerElements } from './configs/registerGUXComponents';
+import './configs/i18n';
+
+registerElements();
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Suspense fallback={<div>Loading i18n</div>}>
```

Comment:
In the unlikely event that `i18next` isn't initialized before the application, this `<Suspense>` component will provide a fallback rendering until they're ready.